src/components/panel/panel.spec.js

```diff
@@ -1059,6 +1059,77 @@ describe('$mdPanel', function() {
     });
   });
 
+  describe('onCloseSuccess logic:', function() {
```

Comment:
I don't think these tests should be it's own group. This functionality is part of the close logic, and therefore should be individual tests after the other close callback tests.
 Like this:
https://github.com/angular/material/blob/master/src/components/panel/panel.spec.js#L877